src/internal/components/configurationPanels/tests/BaseChartConfigurationPanel.spec.tsx

```diff
@@ -0,0 +1,263 @@
+            expect(yAxisSection.props().disabled).toEqual(false);
+        });
+
+        it("should render configuration panel with enable name sections in dual axis chart", () => {
+            const mdObject = {
+                buckets: [
+                    {
+                        localIdentifier: "measures",
+                        items: [
+                            {
+                                measure: {
+                                    localIdentifier: "measureId",
+                                    definition: {
+                                        measureDefinition: {
+                                            item: { uri: "/gdc/md/projectId/obj/9211" },
+                                        },
+                                    },
+                                },
+                            },
+                            {
+                                measure: {
+                                    localIdentifier: "measureId2",
+                                    definition: {
+                                        measureDefinition: {
+                                            item: { uri: "/gdc/md/projectId/obj/9203" },
+                                        },
+                                    },
+                                },
+                            },
+                        ],
+                    },
+                    {
+                        localIdentifier: "view",
+                        items: [
+                            {
+                                visualizationAttribute: {
+                                    localIdentifier: "viewId",
+                                    displayForm: { uri: "/gdc/md/projectId/obj/1024" },
+                                },
+                            },
+                        ],
+                    },
+                ],
+                visualizationClass: { uri: "/gdc/md/projectId/obj/75535" },
+            };
+
+            const wrapper = createComponent({
+                ...defaultProps,
+                mdObject,
+                axis: "dual",
+                properties: {
+                    controls: {
+                        secondary_yaxis: {
+                            measures: ["measureId2"],
+                        },
+                    },
+                },
+            });
+
+            const axisSections = wrapper.find(NameSubsection);
+
+            const xAxisSection = axisSections.at(0);
+            expect(xAxisSection.props().disabled).toEqual(false);
+
+            const yAxisSection = axisSections.at(1);
+            expect(yAxisSection.props().disabled).toEqual(false);
+
+            const secondaryYAxisSection = axisSections.at(2);
+            expect(secondaryYAxisSection.props().disabled).toEqual(false);
+        });
+
+        it("should render configuration panel with enable X axis name section and disable Y axis name section in single axis chart", () => {
```

Comment:
is it `... enabled ... disabled ...`?
also apply for other tests :)

Reply:
Fixed